merfi/iso.py

```diff
@@ -36,7 +39,14 @@ def parse_args(self):
         self.source = util.infer_path(parser.unknown_commands)
         self.check_dependency()
         self.make_iso()
+        self.make_sha256sum()
 
     def make_iso(self):
         cmd = ['genisoimage', '-r', '-o', self.output, self.source]
         util.run(cmd)
+
+    def make_sha256sum(self):
+        with open(self.output, 'rb') as isof:
```

Comment:
Oh, I didn't know I could update the checksum in chunks, that's awesome. Much better.

Re: `os.path.join()`, I really want `+` here, because the string is a new filename, right?

```
>>> import os
>>> print 'test.iso' + '.SHA256SUM'
test.iso.SHA256SUM
>>> print os.path.join('test.iso', '.SHA256SUM')
test.iso/.SHA256SUM
```